File: Java/hayaa.basemodel/src/main/java/hayaa/basemodel/model/BaseFunctionOpenResult.java
package hayaa.basemodel.model;

import java.io.Serializable;

public class BaseFunctionOpenResult implements Serializable {
    public BaseFunctionOpenResult()
    {
        this.ActionResult = true;
    }
    public BaseFunctionOpenResult(boolean actionResult) {
        this.ActionResult = actionResult;
    }

    private boolean ActionResult;
    private int ErrorCode;
    private String ErrorMsg;

    public boolean isActionResult() {
        return ActionResult;
    }

    public void setActionResult(boolean actionResult) {
        ActionResult = actionResult;
    }

    public int getErrorCode() {
        return ErrorCode;
    }

    public void setErrorCode(int errorCode) {
        ErrorCode = errorCode;
    }

    public String getErrorMsg() {
        return ErrorMsg;
    }

    public void setErrorMsg(String errorMsg) {
        ErrorMsg = errorMsg;
    }


}

File: Hayaa.BaseModel/Hayaa.BaseModel/Model/BaseFunctionOpenResult.cs
﻿using System;
using System.Collections.Generic;
using System.Text;

namespace Hayaa.BaseModel
{
    /// <summary>
    /// 函数执行过程结果类
    /// </summary>
    [Serializable]
    public class BaseFunctionOpenResult
    {
        public BaseFunctionOpenResult()
        {
            this.ActionResult = true;           
        }
        public BaseFunctionOpenResult(bool actionResult)
        {
            this.ActionResult = actionResult;          
        }
        /// <summary>
        /// 函数执行过程true表示正常，false表示有异常
        /// </summary>
        public bool ActionResult { set; get; }       
        /// <summary>
        /// 函数异常码
        /// </summary>
        public int ErrorCode { set; get; }
        /// <summary>
        /// 函数执行异常返回信息
        /// </summary>
        public string ErrorMsg { set; get; }
       
    }
}
